### PR TITLE
Chore/143412447/integrate coveralls

### PR DESCRIPTION
**Pull request by Ewanjiru**
**What does this PR do?**

It involves integration of coverall and will update the package.json scripts adding coveralls command, the travis.yml file by adding the npm run coveralls command and the readme file by adding the badge.

**Description of Task to be completed?**

Integrating Coveralls to the project repo and adding a coveralls badge.

**How should this be manually tested?**

Edit any file then commit and make a push to repo.
Login into coveralls.io and locate your repo to see the coverage.

**What are the relevant pivotal tracker stories?**

https://www.pivotaltracker.com/story/show/143412447

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ c4h_keys.txt
 .idea/vcs.xml
 Project_Default.xml
 .env
+yarn.lock
```
Comment (gngchrs) [inline .gitignore]: minor: Should not be ignored

**Comment by muchai-mercy** [comment]
Reviewed

**Review by muchai-mercy — COMMENTED** [review]
seen the coverage on login into coveralls.io and locating the repo.

File: .travis.yml [hunk 2]
```diff
@@ -5,3 +5,5 @@ services:
  - mongodb
 env:
  - MONGOHQ_URL=mongodb://127.0.0.1/reactors
+after_script:
+  - npm run coveralls
```
Comment (gngchrs) [inline .travis.yml:9]: nice:

**Review by gngchrs — APPROVED** [review]
Lgtm

**Review by muchai-mercy — COMMENTED** [review]
Coverage visible

**Pull request merged**